src/main/java/seedu/foodrem/ui/ItemCard.java

```diff
@@ -45,10 +45,12 @@ public ItemCard(Item item, int displayedIndex) {
         super(FXML);
         this.item = item;
         id.setText(displayedIndex + ". ");
-        name.setText(item.getNameToListView());
-        quantityAndUnit.setText(item.getQuantityAndUnitToListView());
-        bought.setText(item.getBoughtDateToListView());
-        expiry.setText(item.getExpiryDateToListView());
+        name.setText(String.valueOf(item.getName()));
+        quantityAndUnit.setText(item.getQuantity() + " " + item.getUnit());
+        bought.setText(String.format("(Bought Date: %s)",
```

Comment:
Hmm, actually may I ask why the formatting was removed from its abstraction (e.g. item.getBoughtDateToListView()) and placed here? I was thinking of the Tell-Don't-Ask principle here, since this could be done in e.g. item.getXXXToListView() methods